From 3f7a5f601e73491ac7c39cc848fa98fe6224978c Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Mon, 16 Aug 2021 21:19:39 -0400
Subject: [PATCH 1/1] tests/Makefile.am: disable test-fgesv.

This test can fail on Gentoo, and has been reported upstream. In the
meantime, having it fail on end users is no good, so we disable it.

Gentoo-bug: https://bugs.gentoo.org/807100
Upstream-bug: https://github.com/linbox-team/fflas-ffpack/issues/341
---
 tests/Makefile.am | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index 62e92bc..83cf71f 100755
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -63,7 +63,6 @@ BASIC_TESTS =               \
 		test-io      \
 		test-maxdelayeddim \
 		test-solve \
-		test-fgesv             \
 		test-simd \
 		test-fgemv \
 		test-nullspace \
@@ -143,7 +142,6 @@ test_ftrtri_SOURCES            = test-ftrtri.C
 #  test_fullranksubmatrix_SOURCES = test-fullranksubmatrix.C
 #  test_invert_SOURCES            = test-invert.C
 #  test_krylov_elim_SOURCES       = test-krylov-elim.C
-test_fgesv_SOURCES             = test-fgesv.C
 #  test_frobenius_SOURCES         = test-frobenius.C
 test_nullspace_SOURCES = test-nullspace.C
 test_fdot_SOURCES = test-fdot.C
-- 
2.31.1

